packages/mui-material/src/Chip/Chip.js

```diff
@@ -90,6 +90,7 @@ const ChipRoot = styled('div', {
       alignItems: 'center',
       justifyContent: 'center',
       height: 32,
+      lineHeight: 1.5,
```

Comment:
```suggestion
      lineHeight: 'normal',
```
I think to keep the same visual, it should be explicitly set as `normal`.